lib/rijksmus-api/client.rb

```diff
@@ -15,24 +15,17 @@ def initialize(search_term, api_token)
         search_term || raise('No search term provided')
       end
 
-
-      def images 
+      def images_response
         page = nil
         endpoint = '/collection'
         options = { query: { q: search_term, imgonly: true, p: page, ps: 100, culture: 'en', key: api_token } }
-        response = self.class.get(endpoint, options)
-        response
+        self.class.get(endpoint, options)
```

Comment:
Actually I forgot to write about this refactoring. As this line is the last of or method is the one that we will have back when we call it. This is the reason why we don't need to pass a variable and then call it in the end. We save one line code and then we rename the method as images response :wink:

Reply:
Nice